CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
-## 1.0.0 October 1, 2024
+## 1.1.0 October 25, 2024
```

Comment:
Are we gonna be at like v1.25.0 by the time all the conversions are done? What if we just keep it at v1.0.0 until we actually deploy it to prod?

Reply:
ok sir